lib/matplotlib/cm.py

```diff
@@ -222,7 +222,6 @@ def get_cmap(self, cmap):
 
 @_api.deprecated(
     '3.6',
```

Comment:
Should the version be bumped to 3.7 when the state has changed from pending to non-pending?

Reply:
Oops, yes, that is true.